demo/lib/DemoControls.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable require-jsdoc */
+
+// Get Transformation Controls
+import addCompatibilityControl from './TransformationControls/addCompatibilityControl.js'
+import addDimImagesControl from './TransformationControls/addDimImagesControl.js'
+import addEditPencilsControl from './TransformationControls/addEditPencilsControl.js'
+import addFooterControl from './TransformationControls/addFooterControl.js'
+import addHideRedlinksControl from './TransformationControls/addHideRedlinksControl.js'
+import addLeadIntroductionControl from './TransformationControls/addLeadIntroductionControl.js'
+import addPlatformsControl from './TransformationControls/addPlatformsControl.js'
+import addTableCollapseControl from './TransformationControls/addTableCollapseControl.js'
+import addTextDirectionControl from './TransformationControls/addTextDirectionControl.js'
+import addThemesControl from './TransformationControls/addThemesControl.js'
+// TODO: add addLazyImageLoadControl
+import util from './util.js'
+const { expandIframeHeight } = util
+
+let ClosureManager
+
+const transformationControls = [
```

Comment:
@berndsi I don't know why the sub-sections Platform, Text direction, and Theme doesn't follow this order, but the rest of the controls seems to follow this array order.

Reply:
Ok, fair enough. I was just curious. Doesn't affect your PR.